.github/workflows/test_doc_build.yml

```
@@ -26,4 +26,38 @@ jobs:
+          echo "url=https://github.com/$repo/actions/runs/$run_id/artifacts/$artifact_id" >> "$GITHUB_OUTPUT"
+
+      - name: Comment with documentation link
+        if: steps.artifact_url.outputs.found == 'true'
+        uses: peter-evans/create-or-update-comment@v4
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          issue-number: ${{ github.event.pull_request.number }}
+          body: |
+            ✅ Documentation built:
+            👉 [Download](${{ steps.artifact_url.outputs.url }})
+            Note: You must be logged in to github and a DASDAE member to access the link.  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)